### PR TITLE
Added Draft posts

### PR DESCRIPTION
**Pull request by chrisg32**
When draft is set to true in front matter pages will not be generated or added to the posts lists.

### CODE REVIEW

File: src/Dtos.cs [hunk 1]
```diff
@@ -10,6 +10,11 @@ public interface IFrontMatter
     /// Useful for generating tag pages.
     /// </summary>
     List<string> Tags { get; set; }
+    
+    /// <summary>
+    /// If true, the blog post will not be generated.
+    /// </summary>
+    bool Draft { get; set; }
```
Comment (tesar-tech) [inline src/Dtos.cs]: I am suggesting:
- Name it `IsDraft`, which strongly implies it's a bool.
- Create default implementation: `bool IsDraft => false;`, which doesn't force user to implement IsDraft in their FrontMatter classes..

**Review by tesar-tech — COMMENTED** [review]
Thanks you for the contribution. It is a good idea to have the ability not to publish drafted blog post.

File: src/Dtos.cs (hunk 1 above)
Comment (chrisg32) [reply src/Dtos.cs]: Done. Both excellent suggestions. Thank you sir.

**Comment by chrisg32** [comment]
> Thanks you for the contribution. It is a good idea to have the ability not to publish drafted blog post.

Thank you for BlazorStatic. It is very similar to a project a start for static generation using asp.net. With .Net 8 static SSR I decided revisit it when I stumbled across BlazorStatic. My site was almost plug and play.

BlazorStatic is very elegant and I would like to help out where I can. Thank again.

**Comment by tesar-tech** [comment]
Thanks for the PR. 
I am glad you find BlazorStatic helpful.

If you come across any features akin to the `IsDraft` flag, or if there's something you feel is missing, please feel free to submit another PR or raise an issue. Over time, I'll be posting issues regarding aspects I believe need enhancement or elements that seem absent. However, that's just my take on things. I genuinely value diverse perspectives and am keen on exploring ways to make it more attractive to others. 

f you're interested in discussing further, let me know (contact in bio)

**Pull request merged**